src/odemis/acq/fastem.py

```diff
@@ -33,6 +33,7 @@
 
 from odemis.acq.align.fastem import align, estimate_calibration_time
 from odemis.gui import FG_COLOUR_WARNING
+from odemis.gui.conf.file import AcquisitionConfig
```

Comment:
Ideally, nothing from odemis.acq depends on odemis.gui. So we might want to even reconsider the `FG_COLOUR_WARNING`.

Reply:
It seems to be used here: https://github.com/delmic/odemis/blob/2419a9a7ec2f2e4ae309cdbc0d83c529bc958f0b/src/odemis/gui/cont/project.py#L434

Reply:
Since it is only used as a default value, I think it should be fine to update the default of the input argument to `"#FFA300"`. Do you agree @nandishjpatel? I can update this in a separate PR
```
def __init__(self, name, coordinates, colour="#FFA300"):
```

Reply:
> Since it is only used as a default value, I think it should be fine to update the default of the input argument to `"#FFA300"`. Do you agree @nandishjpatel? I can update this in a separate PR
> 
> ```
> def __init__(self, name, coordinates, colour="#FFA300"):
> ```

That's fine